src/pineko/evolve.py

```diff
@@ -157,7 +153,7 @@ def evolve_grid(
     sv_method = None
     if operators.operator_card.configs.scvar_method is not None:
         sv_method = operators.operator_card.configs.scvar_method.name
-    xif = 1.0 if (sv_method == "EXPANDED" or sv_method == "EXPONENTIATED") else xif
+    xif = 1.0 if sv_method in ["EXPANDED", "EXPONENTIATED"] else xif
```

Comment:
```suggestion
    xif = 1.0 if operators.operator_card.configs.scvar_method is not None else xif
```

The check with `None` is definitely sufficient, because an `Optional[ScVarMethod]` can only be three values:
- `None`
- `ScVarMethod.expanded`
- `ScVarMethod.exponentiated`

You can also drop completely the `sv_method` variable from this function.